src/ert/scheduler/scheduler.py

```diff
@@ -15,7 +16,6 @@
 from websockets import Headers
 from websockets.client import connect
 
-from ert.async_utils import background_tasks
```

Comment:
This function in `ert.async_utils` is only used here, so it should probably be purged from async_utils , so this PR does not leave dead code lying around.